skiko/src/jvmMain/cpp/linux/drawlayer.cc

```diff
@@ -6,13 +6,95 @@
 #include <X11/Xresource.h>
 #include <X11/extensions/Xrandr.h>
 #include <cstdlib>
+#include <dlfcn.h>
 #include <unistd.h>
 #include <stdio.h>
-#include <vector>
 #include "jni_helpers.h"
+#include <vector>
 
 typedef GLXContext (*glXCreateContextAttribsARBProc)(Display *, GLXFBConfig, GLXContext, Bool, const int *);
 
+static void* loadXrandr() {
```

Comment:
Guess we may share the code by moving xrandr manipulations to its own file.

Reply:
Done.